go.mod

```diff
@@ -3,9 +3,10 @@ module github.com/openshift/source-to-image
 go 1.13
 
 replace (
-	github.com/containerd/containerd => github.com/containerd/containerd v0.2.10-0.20170808145631-06b9cb351610
-	github.com/docker/docker => github.com/openshift/moby-moby v1.4.2-0.20190308215630-da810a85109d
-	github.com/moby/buildkit => github.com/dmcgowan/buildkit v0.0.0-20170731200553-da2b9dc7dab9
+	github.com/Sirupsen/logrus => github.com/sirupsen/logrus v1.2.0
+	github.com/containerd/containerd => github.com/containerd/containerd v1.3.0
+	github.com/docker/docker => github.com/docker/docker v0.0.0-20180522102801-da99009bbb11
+	golang.org/x/crypto => golang.org/x/crypto v0.0.0-20180904163835-0709b304e793
```

Comment:
`golang.org/x/crypto` has been added in the replace stanza, but is not referenced in the `require` stanza. Why is this necessary?

Reply:
After I did my first changes, I noticed that `make test` didn't work. After I dug into it, I've noticed that buildkit and swarmkit should be updated as well, and this lead to all those dependencies  

Tests use buildkit and swarmkit, which also had to be bumped and adjusted (see 9f3ead4fc4a53bdcfd515e5e9e6e6a2fa7b66c6d for reference).